users/users.go

```diff
@@ -47,10 +46,6 @@ func (u User) Validate() error {
 		return ErrMalformedEntity
 	}
 
-	if len(u.Password) < minPassLen {
```

Comment:
why we dont check password here?

Reply:
two reasons:
- we already check it on other places (validating requests for registering and password changing)
- can't import passRegex from api package here

Reply:
@blokovi I would check it here as it was done before and simply do: 
```
func (req userReq) validate() error {
	return req.user.Validate()
}
```

Reply:
@manuio  can't use `passRegex` from api package here in users (because api imports users)

Reply:
@blokovi first remove it from the api layer :)